tcms/rpc/api/forms/testcase.py

```diff
@@ -14,3 +15,5 @@ class UpdateForm(UpdateModelFormMixin, forms.ModelForm):
     class Meta:
         model = TestCase
         exclude = ('tag', 'component', 'plan')  # pylint: disable=modelform-uses-exclude
+
+    default_tester = UserField()
```

Comment:
The same should apply to author and reviewer fields although they are not needed for subsequent tasks.